src/auth/service/auth.service.ts

```diff
@@ -20,37 +19,17 @@ export class AuthService {
     private mailService: MailService,
   ) {}
 
-  async signUp(dto: SignupDto) {
+  async sendVerificationCode(dto: UserDetailsDto) {
```

Comment:
Should also generate and send the verification code.

Reply:
We are skipping verification for auth types other than EMAIL

Reply:
We shouldn't. If someone logs in with any social account that as a linked email, it should be able to login with that email

Reply:
Okay well, this part of the flow wasn't discussed earlier.

Reply:
Okay so for this, we can't be sending verification code for all cases. this is because if the user signs up using any oauth application, their email will already be verified as per business logic. So, im resolving this one.

As for the ability to log in using email even if they have logged in using any other oauth platform, that is being implemented